extensions/wikia/TemplateClassification/TemplateClassificationView.php

```diff
@@ -13,14 +13,17 @@ class View {
 	 * @return string
 	 */
 	public function renderTemplateType( $articleId, $user, $fallbackMsg = '' ) {
-		$templateType = ( new \TemplateClassificationMockService() )->getTemplateType( $articleId );
+		$templateType = lcfirst( ( new \TemplateClassificationMockService() )->getTemplateType( $articleId ) );
 		/**
 		 * template-classification-type-unclassified
 		 * template-classification-type-infobox
 		 * template-classification-type-navbox
 		 * template-classification-type-quote
 		 */
-		$templateName = wfMessage( "template-classification-type-{$templateType}" )->escaped();
+		$templateName = wfMessage(
+			"template-classification-indicator",
+			wfMessage( "template-classification-type-{$templateType}" )
```

Comment:
Since this is being escaped as a parameter to the `template-classification-indicator` message, this should have an output method of `->plain()` or `->text()`. If you don't add an explicit output type it will use `->parse()` which means it would be double escaped and is an unnecessary call to the parser.